content/blog/2019-12-09-comp-cert/index.md

```diff
@@ -0,0 +1,111 @@
++++
+title = "CompCert: Formally Verified C Compiler"
+extra.author = "Samuel Thomas & Daniel Weber"
+extra.latex = true
+extra.bio = """
+  [Samuel Thomas](https://sgtpeacock.com) is a senior undergraduate applying to Graduate Schools for programming languages.
+  [Daniel Weber](https://github.com/Dan12) is an MEng student interested in programming languages and distributed systems.
+"""
++++
+
+## Motivation
+
+The primary motivation of this paper is that compilers these days form a base of trust for most modern applications. If the application's code is correct, then compiled executable of that application will also be correct. However, most modern compilers like GCC and LLVM do have bugs, some of which silently miscompile code without emitting any error messages. Most of these bugs occur when the compiler performs transformations and optimization passes over the source program's AST. The goal of CompCert is to create a compiler that will never silently miscompile code. The way CompCert accomplishes this is by formally verifying (i.e. proving) that each compiler pass does not change the original meaning of the program. The formally verified parts of CompCert are written in Coq, which is a proof assistant with a small functional programming langauge.
```

Comment:
> Coq, which is a proof assistant with a small functional programming langauge.

Consider linking to Coq's homepage (and, for that matter, CompCert's)? Also, I think it would more salient to say Coq is "based on the calculus of constructions" or "based on a dependently typed programming language" than just "based on a small functional PL."

Also, please run spell check (watch your "langauge").